JWT.js

```diff
@@ -0,0 +1,61 @@
+const jwt = require("jsonwebtoken");
+require("dotenv").config();
+const Users = require("./models/Users");
+
+const generateAccessToken = (payload) => {
+  return jwt.sign(payload, process.env.JWT_KEY, { expiresIn: "15s" });
```

Comment:
I believe `15s` is too short 
There is no standard value, but at least a couple of minutes or even hours would be more reasonable